src/devices/tuya.ts

```diff
@@ -6054,10 +6054,13 @@ const definitions: Definition[] = [
         },
     },
     {
-        fingerprint: tuya.fingerprint('TS0601', ['_TZE204_sbyx0lm6', '_TZE204_dtzziy1e', '_TZE204_clrdrnya']),
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE204_sbyx0lm6', '_TZE204_clrdrnya']),
```

Comment:
`_TZE204_dtzziy1e` shouldn't be removed here as the device will not be detected as supported anymore.

Reply:
Dully noted, thanks !